src/simtools/reporting/docs_read_parameters.py

```diff
@@ -62,13 +64,36 @@ def model_version(self, model_version):
             )
         self._model_version = model_version
 
-    def _convert_to_md(self, parameter, input_file):
+    def _convert_to_md(self, parameter, parameter_version, input_file):
         """Convert a file to a Markdown file, preserving formatting."""
         input_file = Path(input_file)
+
         output_data_path = Path(self.output_path / "_data_files")
         output_data_path.mkdir(parents=True, exist_ok=True)
         output_file_name = Path(input_file.stem + ".md")
         output_file = output_data_path / output_file_name
+        image_name = f"{self.array_element}_{parameter}_{self.model_version.replace('.', '-')}"
+        image_path = Path(f"../{IMAGE_PATH}/{image_name}")
+
+        if parameter == "camera_config_file" and parameter_version:
+            outpath = Path(io_handler.IOHandler().get_output_directory().parent / "_images")
+            outpath.mkdir(parents=True, exist_ok=True)
+            image_path = Path(f"{outpath}/{input_file.stem.replace('.', '-')}")
+            if not (image_path.with_suffix(".png")).exists():
+                plot_config = {
+                    "file_name": input_file.name,
+                    "telescope": self.array_element,
+                    "parameter_version": parameter_version,
+                    "site": self.site,
+                    "model_version": self.model_version,
+                    "parameter": parameter,
+                }
+
+                plot_pixels.plot(
+                    config=plot_config,
+                    output_file=image_path,
+                    db_config=self.db_config,
+                )
 
         try:
             # First try with utf-8
```

Comment:
Is the reading of utf8/latin different to what simtools.utisl.general.read_file_encoded_in_utf_or_latin provides? If not, maybe simply use that one?